src/uct/ib/dc/dc_mlx5_ep.c

```diff
@@ -1346,8 +1346,6 @@ uct_dc_mlx5_iface_dci_do_dcs_pending_tx(ucs_arbiter_t *arbiter,
     int is_only                = ucs_arbiter_elem_is_only(elem);
     ucs_arbiter_cb_result_t res;
 
-    ucs_assert(!uct_dc_mlx5_iface_is_dci_rand(iface));
```

Comment:
why removed?

Reply:
this function is "uct_dc_mlx5_iface_dci_do_**dcs**_pending_tx" so probably no need to check it again